Desctiption: Use -fPIC instead of -fpic to fix FTBFS on s390x
Author: Laurent Bigonville <bigon@debian.org>

--- a/apol/Makefile.am
+++ b/apol/Makefile.am
@@ -73,7 +73,7 @@ DEPENDENCIES = $(top_builddir)/libqpol/s
 	$(top_builddir)/config.tcl
 
 AM_CXXFLAGS = @DEBUGCXXFLAGS@ @WARNCXXFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@SEFS_CFLAGS@ @APOL_CFLAGS@ @QPOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@SEFS_CFLAGS@ @APOL_CFLAGS@ @QPOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libapol/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ \
 	@SEFS_LIB_FLAG@ @APOL_LIB_FLAG@ @QPOL_LIB_FLAG@
--- a/libapol/src/Makefile.am
+++ b/libapol/src/Makefile.am
@@ -5,7 +5,7 @@ apolso_DATA = libapol.so.@libapol_versio
 apolsodir = $(libdir)
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ -fpic -I$(srcdir)/../include \
+	@QPOL_CFLAGS@ -fPIC -I$(srcdir)/../include \
 	-DAPOL_INSTALL_DIR='"${setoolsdir}"' \
 	-DLIBAPOL_POLICY_INSTALL_DIR='"@selinux_policy_dir@/policy"' \
 	-DLIBAPOL_DEFAULT_POLICY='"@selinux_default_policy@"'
--- a/libapol/swig/java/Makefile.am
+++ b/libapol/swig/java/Makefile.am
@@ -64,7 +64,7 @@ BUILT_SOURCES = apol_wrap.c \
 	SWIGTYPE_p_void.java
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libapol/include
 AM_JFLAGS = @DEBUGJFLAGS@ @WARNJFLAGS@ \
 	-classpath $(top_builddir)/libqpol/swig/java/qpol.jar
--- a/libapol/swig/python/Makefile.am
+++ b/libapol/swig/python/Makefile.am
@@ -9,7 +9,7 @@ dist_noinst_DATA = $(srcdir)/../apol.i
 BUILT_SOURCES = apol_wrap.c
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libapol/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ \
 	@APOL_LIB_FLAG@ @QPOL_LIB_FLAG@
--- a/libapol/swig/tcl/Makefile.am
+++ b/libapol/swig/tcl/Makefile.am
@@ -10,7 +10,7 @@ dist_noinst_DATA = $(srcdir)/../apol.i
 BUILT_SOURCES = apol_wrap.c
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libapol/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ \
 	@APOL_LIB_FLAG@ @QPOL_LIB_FLAG@
--- a/libpoldiff/src/Makefile.am
+++ b/libpoldiff/src/Makefile.am
@@ -4,7 +4,7 @@ poldiffso_DATA = libpoldiff.so.@libpoldi
 poldiffsodir = $(libdir)
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ \
-	@APOL_CFLAGS@ @QPOL_CFLAGS@ -I$(srcdir)/../include -fpic
+	@APOL_CFLAGS@ @QPOL_CFLAGS@ -I$(srcdir)/../include -fPIC
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@
 
 libpoldiff_a_SOURCES = \
--- a/libpoldiff/swig/java/Makefile.am
+++ b/libpoldiff/swig/java/Makefile.am
@@ -36,7 +36,7 @@ BUILT_SOURCES = poldiff_wrap.c \
 	SWIGTYPE_p_void.java
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libapol/include -I$(top_srcdir)/libpoldiff/include
 AM_JFLAGS = @DEBUGJFLAGS@ @WARNJFLAGS@ \
 	-classpath $(top_builddir)/libqpol/swig/java/qpol.jar:$(top_builddir)/libapol/swig/java/apol.jar
--- a/libpoldiff/swig/python/Makefile.am
+++ b/libpoldiff/swig/python/Makefile.am
@@ -9,7 +9,7 @@ dist_noinst_DATA = $(srcdir)/../poldiff.
 BUILT_SOURCES = poldiff_wrap.c
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libpoldiff/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ \
 	@APOL_LIB_FLAG@ @QPOL_LIB_FLAG@ @POLDIFF_LIB_FLAG@
--- a/libpoldiff/swig/tcl/Makefile.am
+++ b/libpoldiff/swig/tcl/Makefile.am
@@ -10,7 +10,7 @@ dist_noinst_DATA = $(srcdir)/../poldiff.
 BUILT_SOURCES = poldiff_wrap.c
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libpoldiff/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ \
 	@POLDIFF_LIB_FLAG@ @APOL_LIB_FLAG@ @QPOL_LIB_FLAG@
--- a/libqpol/src/Makefile.am
+++ b/libqpol/src/Makefile.am
@@ -11,7 +11,7 @@ BUILT_SOURCES = policy_parse.h
 
 # search in sepol_srcdir/include before system's sepol directory
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ \
-	-I$(srcdir)/../include -I$(sepol_srcdir)/../include @SELINUX_CFLAGS@ -fpic
+	-I$(srcdir)/../include -I$(sepol_srcdir)/../include @SELINUX_CFLAGS@ -fPIC
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@
 
 libqpol_a_SOURCES = \
--- a/libqpol/swig/java/Makefile.am
+++ b/libqpol/swig/java/Makefile.am
@@ -53,7 +53,7 @@ BUILT_SOURCES = qpol_wrap.c \
 	SWIGTYPE_p_void.java
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	-I$(top_builddir) -fpic \
+	-I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libqpol/include
 AM_JFLAGS = @DEBUGJFLAGS@ @WARNJFLAGS@
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ @QPOL_LIB_FLAG@
--- a/libqpol/swig/python/Makefile.am
+++ b/libqpol/swig/python/Makefile.am
@@ -10,7 +10,7 @@ dist_noinst_DATA = $(srcdir)/../qpol.i
 BUILT_SOURCES = qpol_wrap.c
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	-I$(top_builddir) -fpic \
+	-I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libqpol/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ @QPOL_LIB_FLAG@
 
--- a/libqpol/swig/tcl/Makefile.am
+++ b/libqpol/swig/tcl/Makefile.am
@@ -10,7 +10,7 @@ dist_noinst_DATA = $(srcdir)/../qpol.i
 BUILT_SOURCES = qpol_wrap.c
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	-I$(top_builddir) -fpic \
+	-I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libqpol/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ @QPOL_LIB_FLAG@ @TCL_LIB_SPEC@
 
--- a/libseaudit/src/Makefile.am
+++ b/libseaudit/src/Makefile.am
@@ -4,7 +4,7 @@ seauditso_DATA = libseaudit.so.@libseaud
 seauditsodir = $(libdir)
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@APOL_CFLAGS@ @QPOL_CFLAGS@ @XML_CFLAGS@ -I$(srcdir)/../include -fpic
+	@APOL_CFLAGS@ @QPOL_CFLAGS@ @XML_CFLAGS@ -I$(srcdir)/../include -fPIC
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@
 
 libseaudit_a_SOURCES = \
--- a/libseaudit/swig/java/Makefile.am
+++ b/libseaudit/swig/java/Makefile.am
@@ -33,7 +33,7 @@ BUILT_SOURCES = seaudit_wrap.c \
 	SWIGTYPE_p_void.java
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libapol/include -I$(top_srcdir)/libseaudit/include
 AM_JFLAGS = @DEBUGJFLAGS@ @WARNJFLAGS@ \
 	-classpath $(top_builddir)/libqpol/swig/java/qpol.jar:$(top_builddir)/libapol/swig/java/apol.jar
--- a/libseaudit/swig/python/Makefile.am
+++ b/libseaudit/swig/python/Makefile.am
@@ -9,7 +9,7 @@ dist_noinst_DATA = $(srcdir)/../seaudit.
 BUILT_SOURCES = seaudit_wrap.c
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libseaudit/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ \
 	@APOL_LIB_FLAG@ @QPOL_LIB_FLAG@ @SEAUDIT_LIB_FLAG@ @XML_LIBS@
--- a/libseaudit/swig/tcl/Makefile.am
+++ b/libseaudit/swig/tcl/Makefile.am
@@ -10,7 +10,7 @@ dist_noinst_DATA = $(srcdir)/../seaudit.
 BUILT_SOURCES = seaudit_wrap.c
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libseaudit/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ \
 	@SEAUDIT_LIB_FLAG@ @APOL_LIB_FLAG@ @QPOL_LIB_FLAG@ @XML_LIBS@
--- a/libsefs/src/Makefile.am
+++ b/libsefs/src/Makefile.am
@@ -4,9 +4,9 @@ sefsso_DATA = libsefs.so.@libsefs_versio
 sefssodir = $(libdir)
 
 AM_CFLAGS = @DEBUGCFLAGS@ @WARNCFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ @APOL_CFLAGS@ @SQLITE3_CFLAGS@ -I$(srcdir)/../include -fpic
+	@QPOL_CFLAGS@ @APOL_CFLAGS@ @SQLITE3_CFLAGS@ -I$(srcdir)/../include -fPIC
 AM_CXXFLAGS = @DEBUGCXXFLAGS@ @WARNCXXFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ @APOL_CFLAGS@ @SQLITE3_CFLAGS@ -I$(srcdir)/../include -fpic
+	@QPOL_CFLAGS@ @APOL_CFLAGS@ @SQLITE3_CFLAGS@ -I$(srcdir)/../include -fPIC
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@
 
 libsefs_a_SOURCES = \
--- a/libsefs/swig/java/Makefile.am
+++ b/libsefs/swig/java/Makefile.am
@@ -28,7 +28,7 @@ BUILT_SOURCES = sefs_wrap.cc \
 	SWIGTYPE_p_void.java
 
 AM_CXXFLAGS = @DEBUGCXXFLAGS@ @WARNCXXFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libapol/include -I$(top_srcdir)/libsefs/include
 AM_JFLAGS = @DEBUGJFLAGS@ @WARNJFLAGS@ \
 	-classpath $(top_builddir)/libqpol/swig/java/qpol.jar:$(top_builddir)/libapol/swig/java/apol.jar
--- a/libsefs/swig/python/Makefile.am
+++ b/libsefs/swig/python/Makefile.am
@@ -9,7 +9,7 @@ dist_noinst_DATA = $(srcdir)/../sefs.i
 BUILT_SOURCES = sefs_wrap.cc
 
 AM_CXXFLAGS = @DEBUGCXXFLAGS@ @WARNCXXFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libsefs/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ \
 	@APOL_LIB_FLAG@ @QPOL_LIB_FLAG@ @SEFS_LIB_FLAG@ @XML_LIBS@
--- a/libsefs/swig/tcl/Makefile.am
+++ b/libsefs/swig/tcl/Makefile.am
@@ -10,7 +10,7 @@ dist_noinst_DATA = $(srcdir)/../sefs.i
 BUILT_SOURCES = sefs_wrap.cc
 
 AM_CXXFLAGS = @DEBUGCXXFLAGS@ @WARNCXXFLAGS@ @PROFILECFLAGS@ @SELINUX_CFLAGS@ \
-	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fpic \
+	@QPOL_CFLAGS@ @APOL_CFLAGS@ -I$(top_builddir) -fPIC \
 	-I$(top_srcdir)/libsefs/include
 AM_LDFLAGS = @DEBUGLDFLAGS@ @WARNLDFLAGS@ @PROFILELDFLAGS@ @TCL_LIB_SPEC@ \
 	@SEFS_LIB_FLAG@ @APOL_LIB_FLAG@ @QPOL_LIB_FLAG@
